docs/cli/cmd-options.mdx

```diff
@@ -313,7 +317,11 @@ Use `-1` to fetch Child Workflows at any depth.
 
 ## description
 
-Namespace description.
+Namespace description or Endpoint description in markdown format (encoded using the configured Codec Server).
```

Comment:
```suggestion
Namespace description or Endpoint description.
You may use markdown formatting in the description, encoded with the configured Codec Server.
```

Reply:
All of these suggestions should come in PR form to the CLI since this is all generated from that repo. Let's solve this at the source.

Reply:
@bergundy here you go:
- https://github.com/temporalio/cli/pull/693